DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -71,15 +71,15 @@ struct qaMatchEff {
   HistogramRegistry histos{"Histos", {}, OutputObjHandlingPolicy::AnalysisObject};
   //
   // Track selections
-  Configurable<bool> isUseTrackSelections{"isUseTrackSelections", false, "Boolean to switch the track selections on/off."};
-  Configurable<bool> isUseAnalysisTrackSelections{"isUseAnalysisTrackSelections", false, "Boolean to switch if the analysis track selections are used. If true, all the Explicit track cuts are ignored."};
+  Configurable<bool> b_useTrackSelections{"b_useTrackSelections", false, "Boolean to switch the track selections on/off."};
```

Comment:
the coding conventions do not want "underscore" in the variable names. I have never been rigid in the review for this file, but since the current variable name follows the convention already, I would not change it